src/main-page/Hero/Hero.tsx

```diff
@@ -0,0 +1,15 @@
+import { Section } from "components/Section/Section";
```

Comment:
sharing my thoughts about directory structure - May it would be beneficial for us to have directory for UI components (`components`) and directory for main page (`main-page`) if we'd need to add smth else here in the future (and in that case we'd create new folder to store the building blocks of pages based on feature routing - new feature/new page -> new directory). Wdyt about that?